### ISSUE TITLE
Word2Vec.py ReluGrad is not finite

### ISSUE THREAD

**Issue by bugulnoz** [tensorflow/tensorflow#977]
Running the tutorial as downloaded from tensorflow.org results in an exception stating that the ReluGrad is not finite.  This happens with the learning rate set to .00001 and concurrent steps set to 1.  Once this exception is thrown, learning stops.  It happens at epoch 0, roughly step 500,000.

**Comment by girving** [comment]
This seems to have fallen through the cracks.  @a-dai: Did you get a chance to glance at this?  @bugulnoz: Is it still an issue?

**Comment by aselle** [comment]
Automatically closing because there was no response. Please reopen if it is still an issue.
